src/app/page.tsx

```diff
@@ -1,7 +1,100 @@
+                                    <path d="M15.8546 14.1397C14.6007 10.6048 12.0586 10.0562 9.54469 12.4627C7.82026 14.1128 6.08175 15.751 4.37138 17.4131C2.92211 18.8225 2.83925 20.6954 4.13218 21.9584C5.48453 23.2782 7.41846 23.1885 8.97248 21.7372C9.00219 21.7103 9.03189 21.6819 9.06004 21.6535C10.0293 20.7357 10.847 19.4428 12.0133 19.0124C13.0952 18.6133 14.5507 19.1349 16.1876 19.2814C13.8878 21.3187 11.8945 23.4441 9.53062 25.0823C7.05888 26.7952 3.4521 25.9791 1.48534 23.7057C-0.47986 21.4338 -0.536143 18.0348 1.55882 15.8586C3.71319 13.6211 5.95824 11.4493 8.31429 9.40754C11.1644 6.93833 15.803 7.6767 17.8386 10.7572C18.1934 11.2938 18.2998 11.6824 17.7244 12.1742C17.0834 12.7227 16.5534 13.3909 15.853 14.1382L15.8546 14.1397Z" fill="#178435"/>
+                                  </g>
+                                  <defs>
+                                    <clipPath id="clip0_48_203">
+                                      <rect width="27" height="26" fill="white"/>
+                                    </clipPath>
+                                  </defs>
+                                </svg>
+                              </button>
+
+                              <ul className="dropdown-menu">
+                                  <li>
+                                      <svg xmlns="http://www.w3.org/2000/svg" width="36" height="198" viewBox="0 0 48 198" fill="none">
+                                        <rect width="47.1444" height="197.349" fill="#178435"/>
+                                        <a href="#" target='blank'>
+                                          <path d="M3.56272 50.0594C4.49379 46.3488 5.40484 42.7632 6.27455 39.167C6.35458 38.8345 6.28655 38.3584 6.09447 38.0831C3.18523 33.9097 1.92469 29.2867 2.23949 24.2302C2.83441 14.6917 10.1015 6.49649 19.5749 4.77685C25.3467 3.73017 30.773 4.76621 35.6164 8.09643C41.0867 11.8589 44.3094 17.087 44.859 23.7288C45.582 32.4614 42.0431 39.2043 34.7347 43.9669C32.0269 45.7317 28.9776 46.6521 25.7469 46.9859C22.3027 47.341 18.992 46.8396 15.7613 45.6453C15.4771 45.5402 15.069 45.6227 14.7662 45.7384C11.2487 47.0883 7.7405 48.4621 4.22834 49.828C4.04826 49.8985 3.86152 49.9557 3.56272 50.0594ZM4.8406 48.6244C5.16741 48.5153 5.3795 48.4542 5.58492 48.3744C8.59954 47.2 11.6168 46.0363 14.6208 44.838C15.173 44.6172 15.6332 44.6146 16.1948 44.8353C19.7203 46.2198 23.3912 46.6587 27.0941 45.9246C34.8027 44.3951 40.1477 39.953 42.9355 32.5824C44.2067 29.2216 44.4655 25.7291 43.8266 22.2073C42.9982 17.6482 40.8973 13.7315 37.4198 10.6353C32.2643 6.0443 26.2271 4.42042 19.4615 5.70516C15.2957 6.49649 11.7115 8.48212 8.80363 11.5769C3.94822 16.7425 2.20081 22.8497 3.46801 29.7988C4.01491 32.8032 5.3008 35.507 7.10557 37.9714C7.24563 38.1629 7.37635 38.4648 7.32833 38.6736C7.04154 39.9424 6.69606 41.1979 6.37992 42.46C5.87437 44.4696 5.3755 46.4792 4.8406 48.6231V48.6244Z" fill="white"/>
```

Comment:
SVGs and Accessibility Consider extracting SVGs into separate components or using an image asset management library.